docs/data/material/experimental-api/css-theme-variables/customization.md

```diff
@@ -0,0 +1,298 @@
+
+- `createTheme()` returns a theme for `ThemeProvider`.
+- `extendTheme()` returns a theme for `CssVarsProvider`.
+  :::
+
+### Color schemes
+
+The major difference from the default approach is the palette customization. With the new `extendTheme` API, you can specify the palette for all color schemes at once (`light` and `dark` are built-in color schemes) under the `colorSchemes` node.
+
+Here is an example of customizing the `primary` palette:
+
+```js
+import { pink } from '@mui/material/colors';
+
+const theme = extendTheme({
+  colorSchemes: {
+    light: {
+      palette: {
+        primary: {
+          main: pink[600],
+        },
+      },
+    },
+    dark: {
+      palette: {
+        primary: {
+          main: pink[400],
+        },
+      },
+    },
+  },
+});
+```
+
+### Components
+
+[Component customization](/material-ui/customization/theme-components/) remains the same. We recommend to use the value from `theme.vars.*` whenever possible for better debugging experience:
+
+```js
+const theme = extendTheme({
+  components: {
+    MuiChip: {
+      styleOverrides: {
+        root: ({ theme, ownerState }) => ({
+          ...(ownerState.variant === 'outlined' &&
+            ownerState.color === 'primary' && {
+              // this is the same as writing:
+              // backgroundColor: 'var(--mui-palette-background-paper)',
+              backgroundColor: theme.vars.palette.background.paper,
+            }),
+        }),
+      },
+    },
+  },
+});
+```
+
+### Channel tokens
+
+A channel token is a variable that consists of [color space channels](https://www.w3.org/TR/css-color-4/#color-syntax) but without the alpha component. The value of a channel token is separated by a space, e.g. `12 223 31`, which can be combined with the [color functions](https://www.w3.org/TR/css-color-4/#color-functions) to create a translucent color.
+
+The `extendTheme()` automatically generates channel tokens that are likely to be used frequently from the theme palette. Those colors are suffixed with `Channel`, for example:
+
+```js
+const theme = extendTheme();
+const light = theme.colorSchemes.light;
+
+console.log(light.palette.primary.mainChannel); // '25 118 210'
+// This token is generated from `theme.colorSchemes.light.palette.primary.main`.
+```
+
+You can use the channel tokens to create a translucent color like this:
+
+```js
+const theme = extendTheme({
+  components: {
+    MuiChip: {
+      styleOverrides: {
+        root: ({ theme, ownerState }) => ({
+          ...(ownerState.variant === 'outlined' &&
+            ownerState.color === 'primary' && {
+              backgroundColor: `rgba(${theme.vars.palette.primary.mainChannel} / 0.12)`,
+            }),
+        }),
+      },
+    },
+  },
+});
+```
+
+:::warning
+Don't use comma(`,`) as a separator because the result is not a valid CSS color:
```

Comment:
There will be a single comma in `rgba(... , 0.12)`. Is a singular comma more precise?

Reply:
Also: why would a developer use a comma here? Is it because that's what you would normally do if you were writing vanilla CSS? If so, we might want to state that more explicitly. Something like: 

```suggestion
When writing CSS, you would normally separate these values with a comma (`,`). 
In this context, however, you must use a slash (`/`) instead:
```

Reply:
I assume that developers are more familiar with using a comma (`,`) even though it is considered a [legacy syntax](https://www.w3.org/TR/css-color-4/#rgb-functions).

Adding a callout here to make sure that they know why using a comma does not work.